docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -231,29 +231,34 @@ V&V Checks:
     -    
   * - 3
     - % of each facility type have cesarian section capabilities
-    - Public/governmental health facility (62%), private and/or NGO health facility (73%), at home  (0%), other (50%) 
+    - Public/governmental health facility (62%), private for-profit health facility (60%), at home  (0%), private not-for-profit health facility (86%), other (10%)  
```

Comment:
For the value here, the 'other' category is entirely made up, I'm expecting we'll update it based on further research. The 4.8% to 1.8% change was because I am no longer including 'country-specific government health facility' as 'other'.

Reply:
Ok, thanks for the clarification.